### ISSUE TITLE
[250624_OhLoRA_v4] 경량화 Segmentation 모델 데이터셋 준비

### ISSUE THREAD

**Issue by WannaBeSuperteur** [WannaBeSuperteur/AI_Projects#31]
## Project README

- [Project README](https://github.com/WannaBeSuperteur/AI_Projects/blob/main/2025_06_24_OhLoRA_v4/README.md)
- [Schedule Table](https://github.com/WannaBeSuperteur/AI_Projects/blob/main/2025_06_24_OhLoRA_v4/README.md#3-%ED%94%84%EB%A1%9C%EC%A0%9D%ED%8A%B8-%EC%9D%BC%EC%A0%95)

## Task Summary

- **기존 FaceXFormer 모델을 경량화한 Segmentation 모델 필요 (for 실시간 Oh-LoRA 얼굴 출력)**
- Knowledge Distillation 으로 하기 위해서는 **FaceXFormer 결과의 hard label (0 or 1) 이 아닌, soft label 필요**

## Task List

- [ ] Segmentation 학습 입력 데이터 준비 ([StyleGAN-FineTune-v8](https://github.com/WannaBeSuperteur/AI_Projects/blob/P007-001-StyleGAN-VectorFind/2025_05_26_OhLoRA_v3/stylegan/README.md#3-2-fine-tuned-stylegan-stylegan-finetune-v8) 학습에 사용했던 [고품질 여성 얼굴 이미지 4,930 장](https://github.com/WannaBeSuperteur/AI_Projects/blob/P007-001-StyleGAN-VectorFind/2025_05_26_OhLoRA_v3/stylegan/README.md#1-1-%EB%AA%A8%EB%8D%B8-%EA%B5%AC%EC%A1%B0)) (06.24 화, 1d)
- [ ] FaceXFormer 모델 추가 구현 (Knowledge Distillation 으로 하려면, 각 픽셀 별 soft label 정보 필요) (06.24 화 - 06.25 수, 2d)
- [ ] FaceXFormer 모델을 이용하여, 학습 입력 데이터에 대한 출력값 도출 (→ 학습 데이터 완성)	(06.25 수, 1d)
- [ ] Segmentation 학습 데이터 필터링 (Segmentation 결과의 'noise'가 큰 데이터 제거) (06.25 수, 1d)

**Comment by WannaBeSuperteur** [comment]
학습 데이터 준비 완료 (https://github.com/WannaBeSuperteur/AI_Projects/tree/P007-002-Prepare-Data/2025_06_24_OhLoRA_v4)
